resources/templates/base.twig

```diff
@@ -34,6 +33,23 @@
             'lang': {'publish': '{{ lang('publish') }}', 'hide': '{{ lang('hide') }}', 'dropzone': '{{ lang('drop_to_upload') }}'}
         }
     </script>
+    <meta id="embed-title" property="og:site_name" content="{{ config.app_name }}">
+    <meta id="embed-url" property="og:url" content="{{ url }}">
+    <!-- This will only display an image, but that's the most important for now -->
+    {% if type %}
+        <meta name="twitter:card" content="summary_large_image">
+        <meta property="og:type" content="website" />
+        <meta id="embed-title" property="og:title" content="{{ media.filename }} ({{ media.size }})">
+        <meta id="embed-image" property="og:image" content="{{ url }}/raw">
+        <meta id="embed-desc" property="og:description" content="Uploaded at {{ media.timestamp }} by {{ media.username }}">
+        <meta id="discord" name="twitter:image" content="{{ url }}/raw">
+        <meta id="image-src" name="twitter:image:src" content="{{ url }}/raw">
+    {% else %}
+        <meta id="embed-title" property="og:title" content="{{ block('title') }}">
+        <meta id="discord" name="twitter:image" content="/static/images/favicon-32x32.png">
+        <meta id="image-src" name="twitter:image:src" content="/static/images/favicon-32x32.png">
+        <meta id="embed-image" property="og:image" content="/static/images/favicon-32x32.png">
+    {% endif %}
```

Comment:
The base template will be always included in each page, so I don't like put some specific code only that works for the public template. I think you should defining it like a block with a default content and the exdending it in the other views

Reply:
The point of the if else is to see if there is a type, if not we just assume it's not a link to content